webview-sys/webview_mshtml.c

```diff
@@ -110,6 +110,9 @@ typedef struct {
   _IServiceProviderEx provider;
 } _IOleClientSiteEx;
 
+typedef BOOL (*SetThreadDpiAwareness)(DPI_AWARENESS_CONTEXT);

```

Comment:
As @parkovski have mentioned, the calling convention is significant on x86. (But does not matter on x86_64.)

```suggestion
typedef BOOL (WINAPI *SetThreadDpiAwarenessContext)(DPI_AWARENESS_CONTEXT);
```